src/CommunityToolkit.Maui.UnitTests/Views/GravatarImageSource/ImageSourceStub.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace CommunityToolkit.Maui.UnitTests.Views.GravatarImageSource;
```

Comment:
If we do need to keep this (not sure if it's needed with `IGravatarImageSource` going) we should probably move it over to the `Mocks` folder.

Reply:
Removed as the interface has gone.